build_tools/web/flutter_service_worker.js.patch

```diff
@@ -0,0 +1,53 @@
+--- flutter_service_worker.js	2023-03-24 19:00:29
```

Comment:
// Patched flutter_service_worker.js splits up the loading of files so they are parallel. 
//  see: https://github.com/dwyl/app/issues/326#issuecomment-1478314967